### PR TITLE
Configuration store logic fixed

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

**Comment by sbespalov** [comment]
@carlspring can you please merge this?

File: strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationFileManager.java [hunk 1]
```diff
@@ -36,6 +36,7 @@ public void store(final Configuration configuration)
         try
         {
             Resource configurationResource = getConfigurationResource();
+            //Check that target resource stored on FS and not under classpath for example
             if (!configurationResource.isFile())
             {
                 LOGGER.warn(String.format("Skip configuration resource store [%s]", configurationResource));
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationFileManager.java]: Hey, would you mind changing the name of this logger to lowercase? Everything else looks great and you can proceed with self-merging it after this minor change request.

Comment (sbespalov) [reply strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationFileManager.java]: ok

Comment (carlspring) [reply strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationFileManager.java]: Thanks!

**Pull request merged**